SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts

```diff
@@ -0,0 +1,130 @@
+import { Component, OnInit, OnDestroy, ViewChild, ElementRef, Input } from '@angular/core';
+import { Geolocation } from '@ionic-native/geolocation/ngx';
+import { Map } from '../../interfaces/map';
+
+declare var google;
+
+@Component({
+  selector: 'app-outdoor-map',
+  templateUrl: './outdoor-map.component.html',
+  styleUrls: ['./outdoor-map.component.scss'],
+})
+export class OutdoorMapComponent implements OnInit, OnDestroy, Map {
+  @Input() data: any;
+  @ViewChild('map', {static: true}) mapElement: ElementRef;
+  map: any;
+  userMarker: any;
+  mapInitialised: boolean = false;
+  currentPos: any;
+  apiKey: string = "AIzaSyA_u2fkanThpKMP4XxqLVfT9uK0puEfRns";
```

Comment:
This should be in the ENV variables!

Reply:
Yes indeed. However, it is not included in this issue. 

Since the api key is already committed (since sprint 2), it can be seen by anyone if they look at previous commits. But yes, we can definitely create an env variables for that and also include building boundaries and floor-plan properties, which should be done in a more suitable issue.